nengo_loihi/hardware/interface.py

```diff
@@ -215,13 +215,26 @@ def _chip2host_monitor(self, probes_receivers):
     def _chip2host_snips(self, probes_receivers):
         assert self.host_socket_connected
 
+        expected_bytes = 4 * self.nengo_io_c2h_count
```

Comment:
The multiplication of size with 4 can be replaced with multiplication with a constant describing 4 e.g. `bytes_per_int32` or something better.

Reply:
Maybe where we define `channel_packet_size`, we also have `channel_bytes_per_element`.

Also, if we rename `PACKET_SIZE` to `PACKET_ELEMENTS` in the snips (as I was musing about above), then we should rename `channel_packet_size` to `channel_packet_elements`.